config/grunt/custom-tasks/init-rc-release.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+/**
+ * helper function
+ * @param {array} 
+ * @returns {bool}
+ */
+
+if (!String.prototype.isInList) {
+	String.prototype.isInList = function() {
+	   let value = this.valueOf();
+	   for (let i = 0, l = arguments.length; i < l; i += 1) {
+		  if (arguments[i] === value) return true;
+	   }
+	   return false;
+	}
+ }
+
+
+
+
+
+module.exports = function( grunt ) {
+	grunt.registerMultiTask(
+		"init-rc-release",
+		"central entry point to set options in one place",
+		function() {
+            const options = this.options({
+                failOnError: true,
+                enableSvnDeploy: false,
+                githubOwner: '',
+                githubRepo: '',
+                alternativeBranch: 'CI-test',
+                alternativeBranchPush: false,
+                enableGithubRelease: false,
+                noBump: false,
+        });
+        
+
+        let owner;
+        let repo;
+        const pluginVersionArg = grunt.option( "plugin-version" );
+        const type = grunt.option( 'type' ) || options.type;
+        const enableSvnDeployArg = grunt.option( 'enableSvnDeploy' ) ;
+        const enableSvnDeploy = (typeof enableSvnDeployArg === 'undefined' ) ? options.enableSvnDeploy: enableSvnDeployArg;
+        const enableSlackArg = grunt.option( 'enableSlack' ) 
+        const enableSlack = (typeof enableSlackArg === 'undefined' ) ? options.enableSlack: enableSlackArg ;
+        const enableGithubReleaseArg = grunt.option( 'enableGithubRelease') 
+        const enableGithubRelease = (typeof enableGithubReleaseArg === 'undefined' ) ? options.enableGithubRelease: enableGithubReleaseArg;
+        const noBumpArg = grunt.option( "no-version-bump" ) 
+        const noBump = (typeof noBumpArg === 'undefined' ) ? options.noBump: noBumpArg ;
+
+
+        const GithubRepository = process.env.GITHUB_REPOSITORY || ''
+        if (GithubRepository === ''){
+            owner = grunt.option( 'owner' ) || options.githubOwner;
+            repo = grunt.option( 'repo' ) || options.githubRepo;
+        } else {
+            var res = GithubRepository.split("/")
```

Comment:
Please rename `res` to `repositoryUrlParts`

Reply:
changed it